### PR TITLE
[new] Added later@1.1.6

### PR DESCRIPTION
**Pull request by MMore**
See https://github.com/bunkat/later

### CODE REVIEW

File: ajax/libs/later/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "license": "MIT",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha test/**/*-test.js"
+  },
+  "npmName": "later",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/later/package.json]: Include all the files under src, but you only added one file `later.min.js` in this PR, make no sense. And we don't need all of them:
```
array/   component.js  core/  end.js         later.js   package.js  start.js
compat/  constraint/   date/  later-core.js  modifier/  parse/
```

Comment (MMore) [inline ajax/libs/later/package.json]: So I just wanted to add files that are really necessary. Do I have to add these files OR change the package.json?

**Comment by MMore** [comment]
Is it now correct @PeterDaveHello?

**Comment by PeterDaveHello** [comment]
Please squash them into only one commit, thanks.

**Comment by MMore** [comment]
I did.

**Comment by PeterDaveHello** [comment]
There is no any git tags in the repo, so that the git auto-update won't work.

File: ajax/libs/later/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "git://github.com/bunkat/later.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bunkat/later.git",
+    "basePath": "",
+    "files": [
+      "later-min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/later/package.json]: There is no `later-min.js` in the repo but `later.min.js`.

**Comment by Piicksarn** [comment]
ping @cdnjs/intern @cdnjs/intern2

**Comment by pvnr0082t** [comment]
`later` had already been hosted in cdnjs. Ref: https://cdnjs.com/libraries/later. I'm going to close this pull request. Thanks for your contribution. @MMore

**Comment by PeterDaveHello** [comment]
cc #5740

**Pull request closed without merging**